docs/model/product-search-via-elasticsearch.md

```diff
@@ -23,16 +23,24 @@ that are located in `src/Shopsys/ShopBundle/Resources/definition/` directory in
 The directory is configured using `%shopsys.elasticsearch.structure_dir%` parameter.
 
 ### Product data export
-No data are automaticly stored in Elasticsearch by "itself". When you store data into a relational database, they are not stored in Elasticsearch.
+No data are automatically stored in Elasticsearch by "itself". When you store data into a relational database, they are not stored in Elasticsearch.
 You have to export data from database into Elasticsearch actively.
 
-Following product attributes are exported into Elasticsearch (i.e. the search is performed on these fields only):
+Following product attributes are exported into Elasticsearch (i.e. the search or filtering is performed on these fields only):
 * name
 * catnum
 * partno
 * ean
 * description
 * short description
+* flags
+* brand
+* categories
+* prices
+* in_stock
+* parameters
+* ordering_priority
+* calculated_selling_denied
```

Comment:
And calculated displayability (or how it is called)?

Reply:
It's not needed as not visible products are not exported to elasticsearch